modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/metric/SqlViewExporterSpiTest.java

```diff
@@ -332,6 +335,80 @@ public void testTransactions() throws Exception {
         assertTrue(res);
     }
 
+    /** */
+    @Test
+    public void testContinuousQuery() throws Exception {
```

Comment:
Is there any reason why this test has a structure differentiating from the testTransactions test?